index/secondary_index.cc

```diff
@@ -106,6 +106,26 @@ sstring target_parser::get_target_column_name_from_string(const sstring& targets
     return targets;
 }
 
+sstring target_parser::get_base_column_name_from_target(const sstring& target) {
```

Comment:
Nit: always consider using a `std::string_view` instead of `sstring`/`std::string` if you pass a string by reference. It's just more flexible (accepts both `sstring` and `std::string`, also other `std::string_view`) and more performant (no need to materialize a string object, e.g. when passing a string literal).